src/server.c

```diff
@@ -3006,6 +3014,9 @@ sds genRedisInfoString(char *section) {
             server.stat_peak_memory,
             peak_hmem,
             mh->peak_perc,
+            server.stat_peak_memory_memkind,
+            peak_memkind_hmem,
+            (float)memkind_malloc_used*100/server.stat_peak_memory_memkind,
```

Comment:
Please consider division by zero here - if server.stat_peak_memory_memkind is 0 You call here Undefined Behaviour